.github/workflows/publish-commit-bottles.yml

```diff
@@ -64,6 +64,7 @@ jobs:
 
       - name: Pull bottles
         env:
+          HOMEBREW_GITHUB_PACKAGES_USER: brewtestbot
```

Comment:
This definitely shouldn't be needed here as it should be uploading to Bintray still at this point.
```suggestion
```